### ISSUE TITLE
Prepare InnoSetup script for 1.4 release

### ISSUE THREAD

**Issue by retailcoder** [rubberduck-vba/Rubberduck#589]
We need to adjust the script with the new DLL's, and see if the add-in can be registered under HKLM (see #458).

Keep in mind, that `AssertClass` and `TestRunner` types exposed for the units testing COM API, and other types exposed for the source control API, cannot be registered per-user and require elevated privileges for installation.

If Rubberduck was just an ordinary VBE add-in that didn't expose a COM API, this would be a different story.

**Comment by rossknudsen** [comment]
@retailcoder So you want the installer to write the Office Addin reg keys to the HKLM hive rather than the HKCU hive?  I think the only way to know whether this is possible is to suck it and see.  The COM registration occurs using the regasm tool - FYI.

**Comment by rossknudsen** [comment]
I've tried to get the HKLM registry setting to work and have not been successful.  If anyone can manually (i.e. creating the key using regedit) then we can modify the installer script to reproduce that.

**Comment by retailcoder** [comment]
Relevant: http://stackoverflow.com/a/5136814/1188513

> There are few failure modes. Other than:

>  - forgetting to use the /codebase option. Required if you don't deploy the assembly to the GAC, something you should not do on your dev machine.

>  - using the wrong version of Regasm.exe. There are two on a 64-bit machine, the Framework64 directory contains the one you have to use if the client code is 64-bit.

>  - running it from a command prompt that is not elevated. Regasm.exe writes to the HKLM hive of the registry, something that UAC actively prevents. That's an issue on Vista and Win7.

**Comment by retailcoder** [comment]
Could it be because of the `Flags: runascurrentuser` flag?

**Comment by rossknudsen** [comment]
I used the installer which added keys to the HKLM hive in what I thought to be the equivalent place.  VBA didn't read those keys (because it didn't list the addin in the Addin Manager).  So I tried manually creating a few keys in different places but couldn't get any of them to work.  Which is why I said if anyone has success finding where the keys need to be created, we can reproduce that in the installer.  So creating the keys isn't the problem.  Its creating them in the right place, if it is even possible to use HKLM.
